docs/src/modules/components/AppNavDrawer.js

```diff
@@ -610,10 +596,10 @@ function AppNavDrawer(props) {
             )}
             {router.asPath.startsWith('/base/') && (
               <ProductIdentifier
-                name="Base"
+                name="MUI Base"
                 metadata="MUI Core"
                 versionSelector={renderVersionSelector([
-                  { text: `v${basePkgJson.version}`, current: true },
+                  { text: `v${basePkgJson.version.replace('.0.0', '')}`, current: true },
```

Comment:
Will it work with versions like *.0.1?

Reply:
@siriwatknp should we maybe extend a bit the width of the drawer instead of this?

Reply:
@michaldudak with `5.0.1`, you won't have `alpha-XX` which works fine in one line.

@mnajdova need to increase the width for `12px` to make it in one line.

@danilo-leal what do you think?

Reply:
About the width increase, sure!
Are we removing the _alpha_ word then?

Reply:
At this point, I don't think so. It should show the version from the package.

Reply:
Ok, great! It seemed like we would, asked for clarification!